src/xdp/xsk.c

```diff
@@ -1756,9 +1829,9 @@ XskFreeRing(
     XSK_KERNEL_RING *Ring
     )
 {
-    ASSERT(
-        (Ring->Size != 0 && Ring->Mdl != NULL && Ring->Shared != NULL) ||
-        (Ring->Size == 0 && Ring->Mdl == NULL && Ring->Shared == NULL));
+    // ASSERT(
+    //     (Ring->Size != 0 && Ring->Mdl != NULL && Ring->Shared != NULL) ||
+    //     (Ring->Size == 0 && Ring->Mdl == NULL && Ring->Shared == NULL));
```

Comment:
Why did you remove this? Plan on adding it back?

Reply:
Fixed